lib/pbench/server/database/models/active_token.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime
+from typing import TypeVar
+
+from sqlalchemy import Column, DateTime, ForeignKey, Integer, String
+from sqlalchemy.orm import relationship
+
+from pbench.server.database.database import Database
+
+AT = TypeVar("AT", bound="ActiveToken")
+
```

Comment:
Does this do anything more than create an alias for `ActiveToken` which we can use inside the definition of that class?  (Couldn't we just use `"ActiveToken"` instead of `AT` at line 37 and dispense with the extra module-level definition?)